owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -242,11 +243,18 @@ class FileListAdapter(
                     view.binding.Filename.text = file.fileName
                 }
                 ViewType.GRID_IMAGE.ordinal -> {
-                    val layoutParams = fileIcon.layoutParams as ViewGroup.MarginLayoutParams
-                    val marginImage = 4
-                    layoutParams.setMargins(marginImage)
-                    layoutParams.height = ViewGroup.LayoutParams.MATCH_PARENT
-                    layoutParams.width = ViewGroup.LayoutParams.MATCH_PARENT
+                    val view = holder as GridImageViewHolder
+                    if (thumbnail == null) {
+                        view.binding.Filename.visibility = View.VISIBLE
```

Comment:
I see this `Filename`'s visibility is never set to `GONE`, maybe it's always visible but when it is a grid image it's just empty (but not invisible) and you don't need this line. In any case, when you need to work with visibility, let's always use `isVisible` property instead of `visibility`